<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cursores</class>
 <widget class="QMainWindow" name="cursores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cursores</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>LISTA</string>
     </attribute>
     <widget class="QPushButton" name="btDelete">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btLookup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>141</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btAdd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="word">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string>a; </string>
      </property>
      <property name="text">
       <string>k</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="byPos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Por posision</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>46</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="btTool">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>161</y>
        <width>27</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Binario</string>
     </attribute>
     <widget class="QPushButton" name="btSetNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Asignar</string>
      </property>
     </widget>
     <widget class="QLabel" name="binaryNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string>0000000000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btPlus">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="number">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>99999999999999; </string>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Suprimir</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>321</width>
        <height>201</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;El problema del ejercicio 2.9 es que siempre avanza aunque borre por lo tanto cuando hay seguidos se los saltan, tampoco toma en consideracion el ultimo. En la pagina de la lista en la esquina inferior derecha esta para probar ambos aunque tiene cierta modificacion para que se adaptarara a mi lista&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="btIgual">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Efectuar</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="expresion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="numDigits">
       <number>6</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuLista">
     <property name="title">
      <string>Lista</string>
     </property>
     <addaction name="actionUnique"/>
     <addaction name="actionAdd"/>
     <addaction name="actionLookup"/>
     <addaction name="actionDelete"/>
    </widget>
    <addaction name="menuLista"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="text">
    <string>add</string>
   </property>
   <property name="toolTip">
    <string>Agregar caracter a la lista</string>
   </property>
  </action>
  <action name="actionLookup">
   <property name="text">
    <string>lookup</string>
   </property>
   <property name="toolTip">
    <string>Busca caracter en la lista</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>delete</string>
   </property>
   <property name="toolTip">
    <string>Borrar caracter de la lista</string>
   </property>
  </action>
  <action name="actionUnique">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>unique</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionSuprimirLibro">
   <property name="text">
    <string>suprimirLibro</string>
   </property>
  </action>
  <action name="actionSuprimirCorregido">
   <property name="text">
    <string>suprimirCorregido</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btAdd</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDelete</sender>
   <signal>clicked()</signal>
   <receiver>actionDelete</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btLookup</sender>
   <signal>clicked()</signal>
   <receiver>actionLookup</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
